### ISSUE TITLE
orders of deleted schedule not going away

### ISSUE THREAD

**Issue by rfx77** [sos-berlin/joc-cockpit#20]
i have deleted a schedule but the orders do not go away from the daily plan. i have canceled the already submitted orders so they are in planned state. when i restart the daily-plan service the orders are submitted again and planed for the next 7 days.

the deleted schedules are not in the inventory and the trash anymore.

to create an new daily plan with overwrite-option also does not help.

version 2.6.1 (Patch level 1585)

this is in the logs when i try to remove the order:

`2023-08-02T06:48:08,607 INFO  qtp363988129-2996    c.s.j.d.c.JOCOrderResourceImpl               - [deleteOrdersFromPlan][getOrderFilter][jobscheduler]missing permissions
`

when i click on the edit icon in the daily plan i get an exception:
```
com.sos.joc.exceptions.DBMissingDataException: Couldn't find the configuration: Scada.Sch.Test
        at com.sos.joc.classes.inventory.JocInventory.getConfiguration(JocInventory.java:644) ~[sos-webservices-joc-global-2.6.1.jar:?]
        at com.sos.joc.classes.inventory.JocInventory.getConfiguration(JocInventory.java:582) ~[sos-webservices-joc-global-2.6.1.jar:?]
        at com.sos.joc.classes.inventory.JocInventory.getConfiguration(JocInventory.java:577) ~[sos-webservices-joc-global-2.6.1.jar:?]
        at com.sos.joc.inventory.impl.common.AReadConfiguration.read(AReadConfiguration.java:40) ~[sos-webservices-joc-2.6.1.jar:?]
        at com.sos.joc.inventory.impl.ReadConfigurationResourceImpl.read(ReadConfigurationResourceImpl.java:27) ~[sos-webservices-joc-2.6.1.jar:?]
        at jdk.internal.reflect.GeneratedMethodAccessor591.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]

```

**Comment by rfx77** [comment]
Update: After redeploying **every** Workflow in the schedule (with update daily plan option) the orders are gone

**Comment by sosap** [comment]
It is the desired behavior that the Daily Plan and its orders are independent from the underlying schedule. Orders are planned and submitted for the period that you specify in the Daily Plan settings.

When you delete a schedule then you are asked if related orders should be updated (=removed) or if they should remain in place. If you do not agree to delete orders at the point in time of deletion of a schedule then we find the behavior that you describe.

**Comment by rfx77** [comment]
i am not getting asked this question (related orders). 
i am asked if i want to update the daily plan (now, start date, no) but it does not update


![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/4c335b75-e5a4-427f-b6d4-d0e1eb304e21)

**Comment by sosap** [comment]
We checked this and cannot confirm your observation.

When you receive the popup window to confirm removal of a schedule then the options to update the Daily Plan are offered. Update of the Daily Plan includes to remove orders from the Daily Plan starting from the indicated date (now or specific date). This operation removes the orders of the schedule from the Daily Plan.

**Comment by rfx77** [comment]
Hi i did a complete Test with Screenshots and it shows two issues.

1) After the first Release of the Schedule the Orders are not in the Daily Plan. You have to release it a second time
2) After Removal of the Schedule the orders stay in the Daily Plan

I can reproduce this on any JS7 Instance. They have all been upgraded from 2.6.0 to 2.6.1 and pached with lates UI Patch

Here are the steps:

1) Daily Plan before
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/6f090efa-5c57-4ecc-93f5-e065a492f8b5)

2) Create and Release Schedule
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/637f4b72-9a21-41c5-8a15-07a3e055cbc9)

3) Schedule not in the Daily Plan (Bug?)
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/6690c032-4b3e-414f-85e2-391331466358)

4) Release Schedule again. Now the Schedule is in the daily Plan
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/500f7c8d-2d76-474a-b66d-48f9932ec13b)

5) Remove Schedule
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/452d4175-5bf9-45b6-a96e-d61ed16d4978)

6) Schedule stays in the Daily Plan for all planned following days (Bug?)
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/8213f0a1-9911-42b6-97a9-1bc9835d6187)
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/9d0f9b68-eebf-425d-b33c-ca93c21e652f)
![image](https://github.com/sos-berlin/joc-cockpit/assets/61699304/39a5f487-a87d-4118-918d-032c09252733)

**Comment by rfx77** [comment]
There is a permission issue in the logs from daily-plan-service  (last line). i am working as root user

`2023-08-03T06:36:22,340 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-03][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=121]
2023-08-03T06:36:22,348 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-04][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=122]
2023-08-03T06:36:22,352 INFO  JControllerProxy-115 c.s.j.d.OrderListSynchronizer      - [manual][creating/submitting][jobscheduler][2023-08-03][stored]total 1 orders
2023-08-03T06:36:22,353 INFO  JControllerProxy-115 c.s.j.d.OrderListSynchronizer      - [manual][submitOrdersToController][jobscheduler][2023-08-03][1 of 1 orders]history added=1(0.001015s)
2023-08-03T06:36:22,354 INFO  JControllerProxy-115 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-03][coupled with proxy]start submitting 1 orders ...
2023-08-03T06:36:22,354 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-05][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=123]
2023-08-03T06:36:22,354 INFO  JControllerProxy-119 c.s.j.d.OrderListSynchronizer      - [manual][creating/submitting][jobscheduler][2023-08-04][stored]total 1 orders
2023-08-03T06:36:22,355 INFO  JControllerProxy-119 c.s.j.d.OrderListSynchronizer      - [manual][submitOrdersToController][jobscheduler][2023-08-04][1 of 1 orders]history added=1(0.000808s)
2023-08-03T06:36:22,356 INFO  JControllerProxy-119 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-04][coupled with proxy]start submitting 1 orders ...
2023-08-03T06:36:22,364 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-06][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=124]
2023-08-03T06:36:22,364 INFO  JControllerProxy-109 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-03][submitted=1][updated orders=1(0.000489s), history=1(0.001366s)]
2023-08-03T06:36:22,365 INFO  JControllerProxy-102 c.s.j.d.OrderListSynchronizer      - [manual][creating/submitting][jobscheduler][2023-08-05][stored]total 1 orders
2023-08-03T06:36:22,367 INFO  JControllerProxy-102 c.s.j.d.OrderListSynchronizer      - [manual][submitOrdersToController][jobscheduler][2023-08-05][1 of 1 orders]history added=1(0.001265s)
2023-08-03T06:36:22,367 INFO  JControllerProxy-102 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-05][coupled with proxy]start submitting 1 orders ...
2023-08-03T06:36:22,369 INFO  JControllerProxy-119 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-04][submitted=1][updated orders=1(0.000359s), history=1(0.001197s)]
2023-08-03T06:36:22,372 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-07][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=125]
2023-08-03T06:36:22,373 INFO  JControllerProxy-124 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-06][stored]total 1 orders
2023-08-03T06:36:22,377 INFO  JControllerProxy-121 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-05][submitted=1][updated orders=1(0.000446s), history=1(0.001213s)]
2023-08-03T06:36:22,381 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-08][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=126]
2023-08-03T06:36:22,381 INFO  JControllerProxy-119 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-07][stored]total 1 orders
2023-08-03T06:36:22,387 INFO  JControllerProxy-119 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-08][stored]total 1 orders
2023-08-03T06:36:22,387 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-09][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=127]
2023-08-03T06:36:22,394 INFO  JControllerProxy-119 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-09][stored]total 1 orders
2023-08-03T06:36:22,394 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-10][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=128]
2023-08-03T06:36:22,400 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-11][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=129]
2023-08-03T06:36:22,400 INFO  JControllerProxy-114 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-10][stored]total 1 orders
2023-08-03T06:36:22,405 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-12][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=130]
2023-08-03T06:36:22,406 INFO  JControllerProxy-114 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-11][stored]total 1 orders
2023-08-03T06:36:22,411 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-13][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=131]
2023-08-03T06:36:22,412 INFO  JControllerProxy-124 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-12][stored]total 1 orders
2023-08-03T06:36:22,417 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-14][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=132]
2023-08-03T06:36:22,417 INFO  JControllerProxy-124 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-13][stored]total 1 orders
2023-08-03T06:36:22,423 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-15][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=133]
2023-08-03T06:36:22,424 INFO  JControllerProxy-124 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-14][stored]total 1 orders
2023-08-03T06:36:22,430 INFO  ForkJoinPool.commonPool-worker-55 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-16][calculated][total 1 orders][submission created=2023-08-03 06:36:22, id=134]
2023-08-03T06:36:22,430 INFO  JControllerProxy-119 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-15][stored]total 1 orders
2023-08-03T06:36:22,437 INFO  JControllerProxy-124 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-16][stored]total 1 orders
**2023-08-03T06:37:33,477 INFO  JControllerProxy-124 c.s.j.d.c.JOCOrderResourceImpl     - [deleteOrdersFromPlan][getOrderFilter][jobscheduler]missing permissions**
`

**Comment by rfx77** [comment]
I did some additional testing.

1) That the orders does not go into daily-plan on the **first releasing** happens only on one of my instances
2) That the orders are not removed from the daily plan on deletion of the schedule happens **on all** of my instances

When i first release a schedule and it does not get into daily plan this ist the content of the logfiles (further below is the log of the second release which is successful):

**First Release:**

`audit.log:2023-08-03T07:27:18,836 INFO  REQUEST: ./inventory/release - USER: root - PARAMS: {"auditLog":{},"addOrdersDateFrom":"now","update":[{"objectType":"SCHEDULE","path":"/99_Test/Test.Test01"}]} - COMMENT: - - TIMESPENT: - - TICKET: -
service-dailyplan.log:2023-08-03T07:27:18,863 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-03][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,866 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-04][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,868 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-05][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,869 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-06][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,871 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-07][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,873 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-08][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,875 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-09][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,876 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-10][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,878 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-11][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,880 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-12][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,882 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-13][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,884 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-14][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,886 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-15][skip]found 0 schedules
service-dailyplan.log:2023-08-03T07:27:18,887 INFO  qtp363988129-63      c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-16][skip]found 0 schedules
`




**Second Release:**

`audit.log:2023-08-03T07:41:00,804 INFO  REQUEST: ./inventory/release - USER: root - PARAMS: {"auditLog":{},"addOrdersDateFrom":"now","update":[{"objectType":"SCHEDULE","path":"/99_Test/Test.Test01"}]} - COMMENT: - - TIMESPENT: - - TICKET: -
service-dailyplan.log:2023-08-03T07:41:00,817 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-03][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=148]
service-dailyplan.log:2023-08-03T07:41:00,824 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-04][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=149]
service-dailyplan.log:2023-08-03T07:41:00,831 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating/submitting][jobscheduler][2023-08-05][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=150]
service-dailyplan.log:2023-08-03T07:41:00,834 INFO  JControllerProxy-114 c.s.j.d.OrderListSynchronizer      - [manual][creating/submitting][jobscheduler][2023-08-03][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,835 INFO  JControllerProxy-114 c.s.j.d.OrderListSynchronizer      - [manual][submitOrdersToController][jobscheduler][2023-08-03][1 of 1 orders]history added=1(0.000894s)
service-dailyplan.log:2023-08-03T07:41:00,836 INFO  JControllerProxy-114 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-03][coupled with proxy]start submitting 1 orders ...
service-dailyplan.log:2023-08-03T07:41:00,838 INFO  JControllerProxy-180 c.s.j.d.OrderListSynchronizer      - [manual][creating/submitting][jobscheduler][2023-08-04][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,839 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-06][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=151]
service-dailyplan.log:2023-08-03T07:41:00,839 INFO  JControllerProxy-180 c.s.j.d.OrderListSynchronizer      - [manual][submitOrdersToController][jobscheduler][2023-08-04][1 of 1 orders]history added=1(0.000814s)
service-dailyplan.log:2023-08-03T07:41:00,840 INFO  JControllerProxy-180 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-04][coupled with proxy]start submitting 1 orders ...
service-dailyplan.log:2023-08-03T07:41:00,842 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][creating/submitting][jobscheduler][2023-08-05][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,844 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][submitOrdersToController][jobscheduler][2023-08-05][1 of 1 orders]history added=1(0.001293s)
service-dailyplan.log:2023-08-03T07:41:00,845 INFO  JControllerProxy-179 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-05][coupled with proxy]start submitting 1 orders ...
service-dailyplan.log:2023-08-03T07:41:00,846 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-07][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=152]
service-dailyplan.log:2023-08-03T07:41:00,849 INFO  JControllerProxy-114 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-03][submitted=1][updated orders=1(0.000532s), history=1(0.001557s)]
service-dailyplan.log:2023-08-03T07:41:00,849 INFO  JControllerProxy-108 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-06][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,853 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-08][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=153]
service-dailyplan.log:2023-08-03T07:41:00,863 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-09][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=154]
service-dailyplan.log:2023-08-03T07:41:00,865 INFO  JControllerProxy-180 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-07][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,867 INFO  JControllerProxy-172 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-04][submitted=1][updated orders=1(0.000398s), history=1(0.001948s)]
service-dailyplan.log:2023-08-03T07:41:00,867 INFO  JControllerProxy-179 c.s.j.d.OrderApi                   - [manual][addOrdersToController][jobscheduler][2023-08-05][submitted=1][updated orders=1(0.000513s), history=1(0.001375s)]
service-dailyplan.log:2023-08-03T07:41:00,870 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-10][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=155]
service-dailyplan.log:2023-08-03T07:41:00,877 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-08][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,878 INFO  JControllerProxy-172 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-09][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,880 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-11][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=156]
service-dailyplan.log:2023-08-03T07:41:00,884 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-10][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,887 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-12][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=157]
service-dailyplan.log:2023-08-03T07:41:00,889 INFO  JControllerProxy-107 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-11][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,894 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-13][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=158]
service-dailyplan.log:2023-08-03T07:41:00,896 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-12][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,901 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-14][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=159]
service-dailyplan.log:2023-08-03T07:41:00,904 INFO  JControllerProxy-108 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-13][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,909 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-15][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=160]
service-dailyplan.log:2023-08-03T07:41:00,912 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-14][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,917 INFO  ForkJoinPool.commonPool-worker-23 c.s.j.d.DailyPlanRunner            - [manual][creating][jobscheduler][2023-08-16][calculated][total 1 orders][submission created=2023-08-03 07:41:00, id=161]
service-dailyplan.log:2023-08-03T07:41:00,919 INFO  JControllerProxy-107 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-15][stored]total 1 orders
service-dailyplan.log:2023-08-03T07:41:00,928 INFO  JControllerProxy-179 c.s.j.d.OrderListSynchronizer      - [manual][creating][jobscheduler][2023-08-16][stored]total 1 orders
`

**Comment by sosap** [comment]
Let me summarize your findings:

* For a newly created Schedule no orders are added to the Daily Plan when releasing the Schedule. Orders will be added when the Schedule is updated later on.
    * This issue is confirmed and is tracked with
    * [SET-1591: Orders not added to Daily Plan from newly created Schedules](https://change.sos-berlin.com/browse/JOC-1591)
* When deleting a Schedule then orders in the Daily Plan are cancelled (status: submitted -> planned). However, they are not removed but remain in the planned stated due to a permission issue.
    * This issue is confirmed and is tracked with
    * [SET-1592: Orders not removed from Daily Plan if Schedule is deleted](https://change.sos-berlin.com/browse/JOC-1592)
 
Fixes for both issues will become available with releases 2.5.5 and 2.6.2.
